db/models/applications.py

```diff
@@ -40,6 +40,9 @@ class Applications(db.Model):
     last_edited = db.Column("last_edited", DateTime())
 
     def as_dict(self):
+        date_submitted = (
+            self.date_submitted.isoformat() if self.date_submitted else "null"
```

Comment:
Would we ever want to return `"null"`? Maybe it's better to just throw if this value is not found or cannot be converted?

Reply:
When an application is created or updated the date_submitted would be null, its only when we submit the application that we would want to time stamp date_submitted. We could change the return to something like "not yet submit" if thats better?

Reply:
Ah, OK that makes sense then, no need to change.